nifi-docker/dockerhub/README.md

```diff
@@ -62,29 +66,29 @@ The configuration scripts are suitable for at least 1.4.0+.
 
 ## Running a container
 
-### Standalone Instance, Unsecured
+### Standalone Instance secured with HTTPS and Single User Authentication
 The minimum to run a NiFi instance is as follows:
 
     docker run --name nifi \
-      -p 8080:8080 \
+      -p 8443:8443 \
       -d \
       apache/nifi:latest
 
-This will provide a running instance, exposing the instance UI to the host system on at port 8080,
-viewable at `http://localhost:8080/nifi`.
+This will provide a running instance, exposing the instance UI to the host system on at port 8443,
+viewable at `https://localhost:8443/nifi`.
 
 You can also pass in environment variables to change the NiFi communication ports and hostname using the Docker '-e' switch as follows:
 
     docker run --name nifi \
-      -p 9090:9090 \
+      -p 9443:9443 \
```

Comment:
Do we want this now to be 8443?

Reply:
This is an example of overriding the default (8443), I think this is fine

Reply:
Thanks @ChrisSamo632, I agree, this is part of the example showing an alternative port configuration.